api/v2/checluster_types.go

```diff
@@ -92,6 +91,13 @@ type CheClusterDevEnvironments struct {
 	// Default plug-ins applied to DevWorkspaces.
 	// +optional
 	DefaultPlugins []WorkspaceDefaultPlugins `json:"defaultPlugins,omitempty"`
+	// EditorsDownloadUrls provides a list of custom download URLs for editor binaries.
+	// Each entry contains an editor identifier in the `publisher/name/version` format
+	// and the corresponding download URL.
+	// This field is particularly useful in disconnected or air-gapped environments,
+	// where editors cannot be downloaded from the public internet.
+	// +optional
+	EditorsDownloadUrls []EditorDownloadUrl `json:"editorsDownloadUrls,omitempty"`
```

Comment:
Semantically my main concern is that it is not clear that this field is meant to be used only for local-to-remote JetBrains flow. I believe we should consider at very least clarify it on the property description level e.g. currently this field is meant to be used only for JetBrains editors